tensorflow/core/kernels/mlir_generated/BUILD

```diff
@@ -601,6 +673,25 @@ gpu_kernel_library(
     unroll_factors = "4",
 )
 
+gpu_kernel_library(
+    name = "gpu_abs_kernels_experimental",
+    jit_i64_indexed_for_large_tensors_types = [
+        "i8",
+        "i16",
+        "f16",
+        "f32",
+        "f64",
+        "i64",
+    ],
+    op = "abs",
+    test_tags = [
+        "noasan",  # TODO(b/248071004): JIT-compiled code will not be instrumented.
```

Comment:
Is this needed? If so, is it needed everywhere?

Reply:
It was added priorly and only for this kernel.